docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -139,6 +139,35 @@ EKS Anywhere will add by default.
 Modifying the labels associated with the control plane configuration will cause new nodes to be rolled out, replacing
 the existing nodes.
 
+#### controlPlaneConfiguration.upgradeRolloutStrategy
+Configuration parameters for upgrade strategy.
```

Comment:
Can you add a note somewhere saying `upgradeRolloutStrategy` must be the same cp and all worker nodes